examples/src/main/java/com/expediagroup/sdk/rapid/examples/scenarios/booking/SingleRoomBookScenario.java

```diff
@@ -31,7 +32,7 @@ public void run() {
         // Shopping for properties
         logger.info("Getting property availability for test property: {}", Constants.TEST_PROPERTY_ID);
         ShopService shopService = new ShopService();
-        List<Property> propertyAvailabilityList = shopService.getPropertiesAvailability(this.rapidPartnerSalesProfile);
+        List<Property> propertyAvailabilityList = shopService.getSingleRoomPropertiesAvailability(this.rapidPartnerSalesProfile);
 
         if (propertyAvailabilityList != null && !propertyAvailabilityList.isEmpty()) {
```

Comment:
Same comment for early `if` exit, I know this is from a previous change but it will be nice to update it as well.